torch/onnx/symbolic_helper.py

```diff
@@ -545,7 +545,18 @@ def _is_fp(value):
                 torch.bfloat16,
             )
         else:
-            type = value.type().scalarType()
+            try:
+                type = value.type().scalarType()
+            except Exception:
+                if isinstance(value.type(), torch.ListType):
+                    type = value.type().getElementType().dtype()
+                    return type in (
```

Comment:
Following above, I think with if statements this repeated line can be merged with below.